commands/createfixpullrequests.go

```diff
@@ -178,14 +220,12 @@ func (cfp *CreateFixPullRequestsCmd) openFixingPullRequest(impactedPackage, fixB
 
 	commitMessage := cfp.gitManager.GenerateCommitMessage(impactedPackage, fixVersionInfo.FixVersion)
 	log.Info("Running git add all and commit...")
```

Comment:
Running git add all and commit... --> Running 'git add --all' and 'git commit'...

Let's add the same change in the `openAggregatedPullRequest` function.